chainer/variable.py

```diff
@@ -529,3 +529,22 @@ def __hash__(self):
         return super(Variable, self).__hash__()
 
     __array_priority__ = 200
+
+
+def get_array(arg):
+    """Returns ndarray data.
```

Comment:
I think what "ndarray data" means is difficult to understand for beginners. How about writing as follows:
```
"""Extracts the data array of a :class:`~chainer.Variable`.
```
(I think careful readers understand the spec by reading the argument docs, but it's better to make any part of the document easy to understand and difficult to misunderstand.)